### PR TITLE
LPS-119404 Inline

### PR DESCRIPTION
**Pull request by carolmariaabb**
Related issue: https://issues.liferay.com/browse/LPS-119404

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_form_instance.jsp [hunk 1]
```diff
@@ -96,11 +97,23 @@ FormInstancePermissionCheckerHelper formInstancePermissionCheckerHelper = ddmFor
 											cssClass="table-cell-expand table-title"
 											name="name"
 										>
+
+											<%
+											String errorMessage = "";
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_form_instance.jsp:102]: @marcosapmf, can we use `StringPool.BLANK` instead?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_form_instance_descriptive.jsp [hunk 2]
```diff
@@ -32,21 +32,34 @@ String href = (String)request.getAttribute(WebKeys.SEARCH_ENTRY_HREF);
...
-			<c:when test="<%= hasValidStorageType %>">
+			<c:when test="<%= hasValidDDMFormFields && hasValidStorageType %>">
 				<aui:a cssClass="form-instance-name" href="<%= href %>">
 					<%= HtmlUtil.escape(ddmFormInstance.getName(locale)) %>
 				</aui:a>
 			</c:when>
 			<c:otherwise>
+
+				<%
+				String errorMessage = "";
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_form_instance_descriptive.jsp:48]: @marcosapmf, can we use `StringPool.BLANK` instead?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/form_instance_action.jsp [hunk 3]
```diff
@@ -42,10 +42,7 @@ FormInstancePermissionCheckerHelper formInstancePermissionCheckerHelper = ddmFor
 	</c:if>
 
 	<%
-	boolean hasValidDDMFormFields = ddmFormAdminDisplayContext.hasValidDDMFormFields(formInstance);
-	boolean hasValidStorageType = ddmFormAdminDisplayContext.hasValidStorageType(formInstance);
-
-	boolean isValidForm = hasValidDDMFormFields && hasValidStorageType;
+	boolean isValidForm = ddmFormAdminDisplayContext.hasValidDDMFormFields(formInstance) && ddmFormAdminDisplayContext.hasValidStorageType(formInstance);
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/form_instance_action.jsp:45]: OOOOOOMG! The queen of inline (@carolmariaabb) is back! :princess: :tada:

**Review by natocesarrego — COMMENTED** [review]
Another Leonardo di Caprio pull, guys! Great job! :smile: 

leonardo di caprio

@marcosapmf, please see my two tiny suggestions. :wink:

**Comment by natocesarrego** [comment]
@marcosapmf, I'll close this pull due to conflicts, but feel free to forward it to Brian after check my suggestions.

Thanks.

**Pull request closed without merging**